tracer/tracer.go

```diff
@@ -146,16 +186,62 @@ func (t *Tracer) Flush() error {
 		traces = append(traces, t)
 	}
 
-	_, err := t.transport.Send(traces)
+	_, err := t.transport.SendTraces(traces)
 	return err
 }
 
-// worker periodically flushes traces to the transport.
+func (t *Tracer) FlushServices() error {
+	if !t.enabled || !t.servicesModified {
+		return nil
+	}
+
+	if _, err := t.transport.SendServices(t.services); err != nil {
+		return err
+	}
+	log.Printf("sent %d services", len(t.services))
```

Comment:
This should be debug level material so if we can't do it here with the standard logger... maybe drop it?